nemo/core/classes/modelPT.py

```diff
@@ -35,11 +35,20 @@
 from nemo.utils.app_state import AppState
 from nemo.utils.get_rank import is_global_rank_zero
 
+try:
+    from eff import Archive, Runtimes, Origins, __format_version__
+
+    _EFF_PRESENT_ = True
+except ImportError:
+    _EFF_PRESENT_ = False
+
+
 __all__ = ['ModelPT']
 
 _MODEL_CONFIG_YAML = "model_config.yaml"
 _MODEL_WEIGHTS = "model_weights.ckpt"
 _MODEL_IS_RESTORED = False
+_MODEL_EFF_SAVE = True
```

Comment:
By default this should be False.

Reply:
Leaving as is for now, we will discuss this.